src/components/core/DpEditor/DpEditor.vue

```diff
@@ -357,14 +357,12 @@ import {
   TableRow,
   Underline
 } from 'tiptap-extensions'
-
+import { CleanHtml, Tooltip } from '../../../directives/'
```

Comment:
```suggestion
import { CleanHtml, Tooltip } from '../../../directives'
```
It kind of feels odd to nitpick this, since I was the one adding those slashes in the first place, however, the slash should be removed, sorry.